core/main/src/firebolt/handlers/voice_guidance_rpc.rs

```diff
@@ -34,25 +33,26 @@ use ripple_sdk::{
         device::{
             device_accessibility_data::VoiceGuidanceSettings,
             device_events::{
-                DeviceEvent, DeviceEventCallback, DeviceEventRequest, VOICE_GUIDANCE_CHANGED,
+                DeviceEvent, DeviceEventCallback, DeviceEventRequest,
+                VOICE_GUIDANCE_ENABLED_CHANGED, VOICE_GUIDANCE_SETTINGS_CHANGED,
+                VOICE_GUIDANCE_SPEED_CHANGED,
             },
             device_info_request::DeviceInfoRequest,
             device_peristence::{SetBoolProperty, SetF32Property},
         },
         firebolt::fb_general::{ListenRequest, ListenerResponse},
         gateway::rpc_gateway_api::CallContext,
-        storage_property::{StorageProperty, EVENT_VOICE_GUIDANCE_SPEED_CHANGED},
     },
     extn::extn_client_message::ExtnResponse,
     log::error,
 };
-use serde_json::Value;
+use serde_json::{json, Value};
 
 #[derive(Clone)]
-struct VGEventDecorator {}
+struct VGEnabledEventDecorator {}
```

Comment:
I've renamed some existing structs and fields for clarity, since there are actually several voice guidance settings that can be changed as well as several VG-related event listeners. This aligns with what we've done in Ripple-1.0